parts/kuberneteskubelet.service

```diff
@@ -32,7 +32,7 @@ ExecStart=/usr/bin/docker run \
         --allow-privileged=true \
         --enable-server \
         --enable-debugging-handlers \
-        --config=/etc/kubernetes/manifests \
```

Comment:
Hi @amanohar yes an updated templates.go was generated. I just checked that in. I did run `make build` and `make ci`. For `make ci`, windows tests are failing because the url for the zip file needs to be updated. I can regenerate the windows expected testdata once the windows binaries are uploaded by @JiangtianLi.